packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk

```diff
@@ -38,4 +39,8 @@ makeinstall_target() {
   done
   cp -P .libs/* ${INSTALL}/usr/lib
   cp -P libmediainfo.pc ${INSTALL}/usr/lib/pkgconfig
+
```

Comment:
remove this block as we can simplify the copy to only include the .a (as the .la is a broken symlink)